src/cluster_legacy.c

```diff
@@ -1155,6 +1175,16 @@ static void clusterMsgSendBlockDecrRefCount(void *node) {
     }
 }
 
+static void clusterMsgSendBlockDecrRefCountPublish(void *light_block) {
+    clusterMsgSendBlockLight *msgblock = (clusterMsgSendBlockLight *)light_block;
+    msgblock->refcount--;
+    serverAssert(msgblock->refcount >= 0);
+    if (msgblock->refcount == 0) {
+        server.stat_cluster_links_memory -= msgblock->totlen;
+        zfree(msgblock);
+    }
+}
```

Comment:
Rename `clusterLightMsgSendBlockDecrRefCount` ?

Reply:
Removed to reduce duplication.